htlcswitch/link_test.go

```diff
@@ -263,10 +262,9 @@ func TestChannelLinkSingleHopPayment(t *testing.T) {
 	}
 }
 
-// TestChannelLinkBidirectionalOneHopPayments tests the ability of channel
-// link to cope with bigger number of payment updates that commitment
-// transaction may consist.
-func TestChannelLinkBidirectionalOneHopPayments(t *testing.T) {
+// TestChannelLinkCancelFullCommitment tests the ability for links to cancel
+// forwarded HTLCs once all of their commitment slots are full.
+func TestChannelLinkCancelFullCommitment(t *testing.T) {
```

Comment:
Diff looks difficult. Maybe useful for reviewing if the overflow tests are deleted in a separate commit

Reply:
i moved the new test to different spot in the file so that the addition doesn't collide with the removal, lmk what you think